webapp/src/main/webapp/src/locale/messages.pt.xlf

```diff
@@ -327,6 +333,12 @@
 			</segment>
 		</unit>
 		<unit id="common-sync-msgs">
+			<segment>
```

Comment:
I'm confused here, you might as well use different english words too for this change to make sense

Reply:
Fixed, thanks.